src/screens/LanguageSelect/LanguageSelect.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
```

Comment:
will this screen be showed to the user only once he open the app for the first time ever?

Reply:
Can't say for sure when using emulator and asyncStorage updates on every reload, I think it makes sense to create a separate task to test it on actual android device

Reply:
had a small call with Maryna - decided to raise subtask for this case